### PR TITLE
Algorand Support

### LINKED ISSUE #634: Add support for Algorand blockchain

### PR DESCRIPTION
**Pull request by hewigovens**
Fixes #634

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -7,7 +7,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.1'
+        classpath 'com.android.tools.build:gradle:3.5.0'
```
Comment (vikmeup) [inline android/build.gradle]: @madcake is it ok to upgrade?

Comment (madcake) [reply android/build.gradle]: yes. I updated trust app too.

File: src/proto/Algorand.proto [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    string to_address = 1;
+    uint64 fee = 2;
+    uint64 amount = 3;
+    uint64 first_round = 4;
+    uint64 last_round = 5;
+}
+
+// Input data necessary to create a signed transaction.
+message SigningInput {
+    // netowrk / chain id
```
Comment (vikmeup) [inline src/proto/Algorand.proto]: much nicer to split it 👍

File: src/proto/Algorand.proto [hunk 3]
```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package TW.Algorand.Proto;
+option java_package = "wallet.core.jni.proto";
+
+message TransactionPay {
+    string to_address = 1;
+    uint64 fee = 2;
+    uint64 amount = 3;
+    uint64 first_round = 4;
```
Comment (vikmeup) [inline src/proto/Algorand.proto]: what is first round and last round?

Comment (hewigovens) [reply src/proto/Algorand.proto]: round == block, from my understanding

**Pull request merged**